ci/runTestsuite.ps1

```diff
@@ -33,6 +33,7 @@ function run {
     $output = "transformed.xml"
     
     #nosetests  --all-modules --with-xunit pywinauto/unittests
+    nosetests --nologcapture --exclude=testall --with-xunit --with-coverage --cover-html --cover-html-dir=Coverage_report --cover-package=pywinauto --verbosity=3 pywinauto\fuzzydict.py
     nosetests --nologcapture --exclude=testall --with-xunit --with-coverage --cover-html --cover-html-dir=Coverage_report --cover-package=pywinauto --verbosity=3 pywinauto\unittests
```

Comment:
Seems as the second call to nosetests destroys the previous coverage report. Probably we just have to move the unit tests from ```fuzzydict.py``` to a new ```unittests/test_fuzzydict.py``` module